landlab/components/species_evolution/zone.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Zone functions and class of SpeciesEvolver."""
+from collections import OrderedDict
+from enum import IntEnum, unique
+
+import numpy as np
+
+
+@unique
+class Connection(IntEnum):
+    """Zone connection type.
+
+    Connection type represents the connectivity of zones in two time steps. It
+    is described with the pattern, x-to-y where x and y are the descriptive
+    counts (none, one, or many) of zone(s) at the earlier and later time step,
+    respectively. For example, a connection type of one-to-many is a zone in
```

Comment:
is `one-to-many` a zone in the earlier time step that spatially overlapped ONE ZONE BUT in the later time OVERLAPS MULTIPLE ZONES?

Reply:
I think that might be weird language that I used above, but it shows my confusion. Maybe `one-to-many` actually means one zone broke up into many zones? But then I am unclear how a `none_to_none` would be used, because how can you label a zone that didn't exist prior or currently?

Reply:
The second comment is accurate. `none_to_none` might be useless. Including it did make programming the zone functions more consistent.